demo/src/test/java/com/test/core/keyword/CustomKeyword.java

```diff
@@ -1,151 +1,238 @@
-    }
-
-    /**
-     * Check element is display :v
-     * 
-     * @param element
-     * @return
-     */
-    public boolean isDisplay(WebElement element){
-        return waitForElementIsDisplayed(element).isDisplayed();
-    }
-
-    /**
-     * Wait to element visible
-     * 
-     * @param element to check display and clickable 
-     * @return element to be located
-     */
-    public WebElement waitForElementIsDisplayed(WebElement element){
-        try {
-            return wait.until(ExpectedConditions.elementToBeClickable(element));
-        } catch (WebDriverException e) {
-            throw new WebDriverException("Element is not displayed ");
-        }
-    }
-
-    /**
-     * Check place holder equal given text or not
-     * 
-     * @param element
-     * @return
-     */
-    public boolean checkPlaceHolder(WebElement element, String text){
-        String attribute = waitForElementIsDisplayed(element).getAttribute("placeholder");
-        return (attribute.equals(text));
-    }
-
-    /**
-     * Click to element
-     * 
-     * @param element to click
-     */
-    public CustomKeyword click(WebElement element){
-        Actions actions = new Actions(this.driver);
-        actions.moveToElement(element).build().perform();
-        waitForElementIsDisplayed(element).click();
-        return new CustomKeyword(driver);
-    }
-}
+package com.test.core.keyword;
+
+import java.util.List;
+import org.openqa.selenium.By;
+import org.openqa.selenium.JavascriptExecutor;
+import org.openqa.selenium.WebDriver;
+import org.openqa.selenium.WebDriverException;
+import org.openqa.selenium.WebElement;
+import org.openqa.selenium.interactions.Actions;
+import org.openqa.selenium.support.ui.ExpectedConditions;
+import org.openqa.selenium.support.ui.Select;
+import org.openqa.selenium.support.ui.WebDriverWait;
+
+public class CustomKeyword {
+    private WebDriver driver;
+    private WebDriverWait wait;
+    
+    public CustomKeyword(WebDriver driver, WebDriverWait wait) {
+        this.driver = driver;
+        this.wait = wait;
+    }
+    /**
+     * Keyword for open url with checking if the url valid or not
+     * @param baseUrl
+     * @throws Exception
+     */
+    public void openUrl(String baseUrl) throws Exception {
+        if(baseUrl.startsWith("http://") || baseUrl.startsWith("https://"))
+        {
+            driver.get(baseUrl);
+        }
+        throw new Exception("url not start with https or http. Double check baseUrl");
+    }
+
+    /**
+     * Wait for Element displayed and input text
+     * @param element
+     * @param text
+     */
+    public void sendKeys(WebElement element, String text){
+        try{
+            waitForElementIsDisplayed(element).sendKeys(text);
+        }catch(WebDriverException ex){
+            throw new WebDriverException("Element not availabe to input text!");
+        }
+    }
+
+    public void waitToPageLoad() throws InterruptedException{
+        Thread.sleep(4000);       
```

Comment:
should remove thread.sleep and find the way to handle timeout

Reply:
This review has resolved